.github/workflows/create_release.yml

```
@@ -227,8 +227,8 @@ jobs:
         cp out/bin/wasm-opt* binaryen-$VERSION/
         tar -czf $TARBALL binaryen-$VERSION
         cmake -E sha256sum $TARBALL > $SHASUM
-        echo "::set-output name=tarball::$TARBALL"
-        echo "::set-output name=shasum::$SHASUM"
+        echo "tarball=$tarball" >> $GITHUB_ENV
+        echo "shasum=$shasum" >> $GITHUB_ENV    
```

```suggestion
        echo "shasum=$shasum" >> $GITHUB_ENV
```
These extra spaces also appear above - is there a reason for them?